### DISCUSSION TITLE
Unable to apply for student developer pack, even with proper documentation

### DISCUSSION THREAD

**Discussion by FGHIGHSCHOOL** [community/community#59815] [GitHub Education]
### Hi everyone,

I have provided tried truly everything, my login email is my school-issued email, and my school info is everywhere in my profile, I have tried uploading my student ID which got rejected, and then a diploma proving my advancement to the next year of education, all with proper dates and proof. It keeps rejecting my requests and gives nonsensical reasons for it which are unsolvable in my case, please help and/or fix this, this is very much unacceptable and makes me think it has been done on purpose to discourage people from receiving benefits.

**Comment by samus-aran** [comment]
Hi @FGHIGHSCHOOL  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the latest rejection reasons? This might help someone give you a nudge in the right direction.
